scRNA-seq-advanced/02-dataset_integration.Rmd

```diff
@@ -95,7 +95,7 @@ We want to read in just four of these files, as listed previously.
 To read in these files, we could use the `readr::read_rds()` function (or the base R `readRDS()`) four times, once for each of the files.
 We could also use a `for` loop, which is the approach that many programming languages would lean toward.
 A different and more modular coding approach to reading in these files (and more!) is to leverage the [`purrr`](https://purrr.tidyverse.org/) `tidyverse` package, which provides a convenient set of functions for operating on lists.
-You can read more about these functions and their power and utility in R in [Chapter 21 of _R for Data Science_](https://r4ds.had.co.nz/iteration.html#for-loops-vs.-functionals).
+You can read more about these functions and their power and utility in R in [the _R for Data Science_ e-book](https://r4ds.hadley.nz/iteration.html).
```

Comment:
I'm not sure this page is really the best source any more for the content I was after originally. The older version had more of an explainer about functional programming rather than just the examples. I don't want to leave the link to the old version, but it really was pretty good before!

I'd probably also say "the `purrr` functions" rather than just "these functions" or maybe something slightly different...

Reply:
What about https://adv-r.hadley.nz/functionals.html?
This also seems interesting https://www.r4wrds.com/intermediate/m_iteration

Reply:
The advanced R link is probably the better one there. I'd go with that.